docs/node-mixin/rules/rules.libsonnet

```diff
@@ -16,7 +16,7 @@
             ||| % $._config,
           },
           {
-            // CPU utilisation is % CPU is not idle.
+            // CPU utilisation is % CPU is not idle. This represents CPU saturation.
```

Comment:
Hmm, I guess in the USE way of viewing things, utilisation is utilisation and not saturation.